### DISCUSSION TITLE
AD Smart Card Authentication

### DISCUSSION THREAD

**Discussion by timecoder21** [datalust/seq-tickets#1785] [Help]
I'm trying to configure Seq to be authenticated with Active Directory. However, all AD authentication is by smart card (i.e. we have no traditional passwords). When selecting the Integrated Windows Authentication link on the Seq login page it displays Authentication Failed. No access to this Seq instance has been granted. The Seq log shows an entry that say User successfully authenticated, but no user account exists and automatic provisioning is off". Next entry is Unable to log on user. Smartcard logon is required and was not used. 

What configuration have I missed to make this work?

**Comment by nblumhardt** [comment]
Hi! The first error suggests that you need to add the user to Seq first, via Settings > Users (it's important to format the username as `DOMAIN\user` in this case).

I don't have any direct experience with setting up smart card auth specifically with Seq, but I assume that integrated authentication should work. I'm not sure why it's then falling through to an API that requires local smart card auth. The diagnostic log will include more information (stack traces etc.) - if you can send `support@datalust.co` either the relevant lines from the log file, or a diagnostic report from Settings > Diagnostics, we should be able to figure out what's going wrong.

Hope this helps,
Nick

**Reply by timecoder21** [reply]
Thanks, I'll email support.
